receiver/dockerstatsreceiver/metadata.yaml

```diff
@@ -137,6 +137,20 @@ metrics:
       if_enabled: This metric will be disabled in v0.82.0 and removed in v0.85.0.
     gauge:
       value_type: double
+  container.cpu.limit:
+    enabled: false
+    description: "CPU limit set for the container."
+    unit: "{cpus}"
+    gauge:
+      value_type: double
+  container.cpu.shares:
+    enabled: false
+    description: "CPU shares set for the container."
+    unit: "1"
+    sum:
```

Comment:
I don't understand why `container.cpu.shares` is defined as a Sum. If I'm not mistaken, the CPU shares value can be set by the container operator during container creation, just like `container.cpu.limit` - in that case it should also be a Gauge, shouldn't it?

Reply:
Thanks for spotting it @astencel-sumo 

Fix submitted